spec/Section 4 -- Introspection.md

```diff
@@ -111,14 +111,15 @@ __type(name: String!): __Type
 ```
 
 These fields are implicit and do not appear in the fields list in the root type
-of the query operation.
+of the query operation. They are available even if the schema does not provide
+a query operation of its own.
 
 The schema of the GraphQL schema introspection system:
 
 ```graphql
 type __Schema {
   types: [__Type!]!
-  queryType: __Type!
+  queryType: __Type
```

Comment:
I don't believe this breaking change is necessary; the (implicit) query type is still present in order to perform introspection and querying it with the introspection query `{__typename}` would reveal that.

```suggestion
  queryType: __Type!
```